src/main/java/com/amazonaws/cloudhsm/examples/ECImportKey.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ * software and associated documentation files (the "Software"), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.amazonaws.cloudhsm.examples;
+
+import com.amazonaws.cloudhsm.jce.provider.CloudHsmProvider;
+
+import java.io.IOException;
+import java.security.interfaces.ECPrivateKey;
+import java.security.interfaces.ECPublicKey;
+import java.security.Key;
+import java.security.KeyPair;
+import java.security.KeyFactory;
+import java.security.KeyPairGenerator;
+import java.security.spec.ECPrivateKeySpec;
+import java.security.spec.ECPublicKeySpec;
+import java.security.Security;
+import java.security.spec.ECGenParameterSpec;
+
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyAttribute;
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyAttributesMap;
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyAttributesMapBuilder;
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyPairAttributesMap;
+import com.amazonaws.cloudhsm.jce.provider.attributes.KeyPairAttributesMapBuilder;
+
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+
+import javax.crypto.Cipher;
+
+/**
+ * Generate an EC Key Pair using BouncyCastle then import it into the HSM
```

Comment:
Could you update the description to call out that we are demonstrating two different ways to import keys?
Something like

```
Demonstrate multiple methods of importing an EC Key Pair
 * Import using a KeySpec
 * Import with a custom AttributeMap
```